js/navigation.js

```
@@ -1,23 +1,674 @@
+                break;
+
+            case 'End':
+                event.preventDefault();
+                items[items.length - 1].focus();
+                this.keyboardFocusIndex = items.length - 1;
+                break;
+
+            case 'Escape':
+                event.preventDefault();
+                this.closeAllDropdowns();
+                const trigger = this.activeDropdown?.querySelector('.dropdown-trigger');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)